src/Coalesce.Framework.Persister.Accumulo/src/main/java/com/incadencecorp/coalesce/framework/persistance/accumulo/AccumuloFeatureIterator.java

```diff
@@ -423,16 +424,26 @@ private void populateFeature(SimpleFeature feature, CoalesceRecord record) throw
             String fieldName = normalizer.normalize(recordsetName, field.getName());
             ECoalesceFieldDataTypes dataType = field.getDataType();
 
-            // If there is not a value do not set the attribute.
-            if (field.getBaseValue() != null)
+            if (field.isFlatten())
             {
-                if (field.isListType())
+                if (dataType == ECoalesceFieldDataTypes.FILE_TYPE)
                 {
-                    setFeatureAttribute(feature, fieldName, dataType, field.getBaseValue());
+                    setFeatureAttribute(feature,
+                                        fieldName,
+                                        ECoalesceFieldDataTypes.STRING_TYPE,
+                                        ((CoalesceFileField) field).getFilename());
                 }
-                else
+                else if (field.getBaseValue() != null)
```

Comment:
Should the check for a null BaseValue also happen for FileType?   Why is FILE_TYPE different?